src/style.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+.smallSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+.first_form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.second_form {
+  margin-top: 20px;
+  position: relative;
+}
```

Comment:
Are there any differences between this styles? Use one class for both forms, don't repeat yourself